test/e2e/kubectl/portforward.go

```diff
@@ -667,6 +669,10 @@ var _ = SIGDescribe("Kubectl Port forwarding", func() {
 			err = conn.Close()
 			framework.ExpectNoError(err, "couldn't close local connection")
 
+			ginkgo.By("Close response body")
+			err = resp.Body.Close()
```

Comment:
If I'm looking correctly, the place where we read `resp` is here: https://github.com/kubernetes/kubernetes/blob/67216cfdd980cdd0234866d66a9ffe2ba3d8fcc4/test/e2e/kubectl/portforward.go#L646-L648 

so why not closing it right after that block, it doesn't change logic per se, but is easier to reason about the flow.

Reply:
The block below create a new value for `resp`.

Reply:
That makes sense. Updated.